nuxeo-mongodb-directory/src/main/java/org/nuxeo/directory/mongodb/MongoDBSession.java

```diff
@@ -111,15 +126,54 @@ public DocumentModelList getEntries() throws DirectoryException {
 
     @Override
     public DocumentModel createEntry(Map<String, Object> fieldMap) throws DirectoryException {
+        return createEntry(directoryName, fieldMap);
+    }
+
+    public DocumentModel createEntry(String collection, Map<String, Object> fieldMap) throws DirectoryException {
```

Comment:
Do you really need this method which takes collection name as parameter ? I don't see any uses except `createEntry(Map)`.